dotCMS/src/integration-test/java/com/dotcms/util/MimeTypeUtilsTest.java

```diff
@@ -26,6 +26,14 @@ public static void prepare() throws Exception {
         IntegrationTestInitService.getInstance().init();
     }
 
+    @Test
+    public void test_wrong_mime_MimeType_all(){
+
+        boolean match = MimeTypeUtils.match("*/pdf", "pdf");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'match' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5411758)